### ISSUE TITLE
[Snapshots] Some custom device specs not rendering expected dimensions

### ISSUE THREAD

**Issue by rbro112** [EmergeTools/emerge-android#221]
As reported by a client:
```
@Preview(name = "Phone (Portrait)", device = "spec:width=390dp,height=860dp")
@Preview(name = "Phone (Landscape)", device = "spec:width=430dp,height=860dp,orientation=landscape")
@Preview(name = "Tiny Phone (Landscape)", device = "spec:width=320dp,height=812dp,orientation=landscape")
@Preview(name = "Tablet (Landscape)", device = "id:Nexus 9")
@Preview(name = "Tablet (Portrait)", device = "spec:parent=Nexus 9,orientation=portrait")
```

Some custom specs aren't being rendered as expected.

Expected (based on Android studio):
![image](https://github.com/user-attachments/assets/c9d96c7d-67c4-4ffe-8748-0ee9e8be153d)

Actual:
![image](https://github.com/user-attachments/assets/1725fb59-a244-41b1-b95b-d8d1c32b0f0c)

**Comment by rbro112** [comment]
A temporary workaround for this issue is to add `@IgnoreEmergeSnapshot` from the `snapshots-annotations` dependency. Full details can be found [here](https://github.com/EmergeTools/emerge-android/blob/main/snapshots/README.md#ignoring-previews-from-snapshotting).

**Comment by rbro112** [comment]
Fix has landed for this: https://github.com/EmergeTools/emerge-android/pull/243

Will be cutting a 1.3 release soon that will address this! I'll update this issue once the new version is out.

**Comment by rbro112** [comment]
Upon further investigation, #243 did not properly address the issues.

We've implemented a more comprehensive fix in #285 and will be releasing a 1.3.0 following
